polynote-kernel/src/main/scala/polynote/kernel/PolyKernel.scala

```diff
@@ -4,24 +4,22 @@ import java.io.File
 import java.net.URL
 import java.util.concurrent.{ConcurrentHashMap, Executors}
 
-import cats.effect.{ContextShift, IO}
 import cats.effect.concurrent.{Ref, Semaphore}
 import cats.effect.internals.IOContextShift
+import cats.effect.{ContextShift, IO}
+import cats.syntax.apply._
 import cats.syntax.either._
 import cats.syntax.flatMap._
-import cats.syntax.apply._
-import fs2.Stream
 import fs2.concurrent.{Enqueue, Queue, Topic}
 import org.log4s.{Logger, getLogger}
 import polynote.kernel.PolyKernel.EnqueueSome
 import polynote.kernel.lang.LanguageKernel
-import polynote.kernel.util._
+import polynote.kernel.util.{RuntimeSymbolTable, _}
 import polynote.messages.{Notebook, NotebookCell}
 
-import scala.collection.mutable
 import scala.concurrent.ExecutionContext
-import scala.reflect.internal.util.{AbstractFileClassLoader, BatchSourceFile}
 import scala.reflect.internal.util.ScalaClassLoader.URLClassLoader
+import scala.reflect.internal.util.{AbstractFileClassLoader, BatchSourceFile}
```

Comment:
Looks like my IntelliJ reordered some commits...

What are your import settings? I can switch my IntelliJ to whatever you have so they don't fight each other. I've attached mine. 

![screen shot 2019-02-04 at 10 23 49 am](https://user-images.githubusercontent.com/5430417/52228342-fdb84d00-2866-11e9-8e0d-bba40557897f.png)

Reply:
Mine looks the same...

Reply:
Weird...